### PR TITLE
[REGEDIT] Update Portuguese (pt-PT) translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[REGEDIT] Update Portuguese (pt-PT) translation

### CODE REVIEW

**Review by JoachimHenze — APPROVED** [review]

File: base/applications/regedit/lang/pt-PT.rc [hunk 1]
```diff
@@ -18,50 +18,50 @@ BEGIN
     STRING_USAGE "Usage:\n\
```
Comment (HBelusca) [inline base/applications/regedit/lang/pt-PT.rc:18]: It's also "Usage" in portuguese?

File: base/applications/regedit/lang/pt-PT.rc [hunk 2]
```diff
@@ -18,50 +18,50 @@ BEGIN
...
+                 Se nenhuma chave for especificada, exportar todo o registo.\n\
+  /S             Modo silencioso. Nenhuma mensagem será exibida.\n\
+  /V             Iniciar a GUI no modo avançado. Ignorado.\n\
+  /?             Mostrar estas informações e terminar.\n\
+  [filename]     A localização do ficheiro que contém informações do registo a\n\
+                 ser importado. Quando usada com [/E], esta opção especifica o\n\
+                 local do ficheiro para onde as informações do registo serão exportadas.\n\
+  [reg_key]      A chave do registo a ser modificada.\n\
 \n\
 Usage examples:\n\
```
Comment (HBelusca) [inline base/applications/regedit/lang/pt-PT.rc:38]: Translation?

File: base/applications/regedit/lang/pt-PT.rc [hunk 3]
```diff
@@ -18,50 +18,50 @@ BEGIN
     STRING_USAGE "Usage:\n\
   regedit [options] [filename] [reg_key]\n\n\
 Options:\n\
```
Comment (HBelusca) [inline base/applications/regedit/lang/pt-PT.rc:20]: Translation?

Comment (zecarlos1957) [reply base/applications/regedit/lang/pt-PT.rc]: Yes. I forget. Thanks.

File: base/applications/regedit/lang/pt-PT.rc [hunk 4]
```diff
@@ -15,53 +15,53 @@ END
...
-  /V             Launch the GUI in advanced mode. Ignored.\n\
-  /?             Display this information and exit.\n\
-  [filename]     The location of the file containing registry information to\n\
-                 be imported. When used with [/E], this option specifies the\n\
-                 file location where registry information will be exported.\n\
-  [reg_key]      The registry key to be modified.\n\
+    STRING_USAGE "Uso:\n\
+  regedit [Opções] [Ficheiro] [Chave]\n\n\
+Opções:\n\
+  [sem opções]    Iniciar a versão gráfica deste programa.\n\
```
Comment (JoachimHenze) [inline base/applications/regedit/lang/pt-PT.rc:21]: The last commit introduced this glitch, which was pushed after my initial review.
```suggestion
  [sem opções]   Iniciar a versão gráfica deste programa.\n\
```

File: base/applications/regedit/lang/pt-PT.rc [hunk 5]
```diff
@@ -15,53 +15,53 @@ END
...
-                 If no key is specified, the entire registry is exported.\n\
-  /S             Silent mode. No messages will be displayed.\n\
-  /V             Launch the GUI in advanced mode. Ignored.\n\
-  /?             Display this information and exit.\n\
-  [filename]     The location of the file containing registry information to\n\
-                 be imported. When used with [/E], this option specifies the\n\
-                 file location where registry information will be exported.\n\
-  [reg_key]      The registry key to be modified.\n\
+    STRING_USAGE "Uso:\n\
+  regedit [Opções] [Ficheiro] [Chave]\n\n\
```
Comment (HBelusca) [inline base/applications/regedit/lang/pt-PT.rc:19]: No idea what the usual way is, but the 1st letter of the option names may not need to be uppercased.

**Review by HBelusca — APPROVED** [review]

File: base/applications/regedit/lang/pt-PT.rc (hunk 5 above)
Comment (zecarlos1957) [reply base/applications/regedit/lang/pt-PT.rc]: Maybe you're right. It makes sense

File: base/applications/regedit/lang/pt-PT.rc [hunk 6]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     RegEdit for ReactOS
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
```
Comment (binarymaster) [inline base/applications/regedit/lang/pt-PT.rc:3]:
```suggestion
 * PROJECT:     ReactOS Registry Editor
 * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
```

File: base/applications/regedit/lang/pt-PT.rc [hunk 7]
```diff
@@ -1,3 +1,10 @@
+/*
+ * PROJECT:     ReactOS Registry Editor
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
```
Comment (binarymaster) [inline base/applications/regedit/lang/pt-PT.rc:5]:
```suggestion
 * TRANSLATORS: Copyright 2003 Marcelo Duarte <marcelotduarte@gmail.com>
 *              Copyright 2004 Sylvain Pétréolle <spetreolle@yahoo.fr>
 *              Copyright 2019-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
 *              Copyright 2021 Gabriel Aguiar <fgsoftwarestudio@gmail.com>
```

**Pull request merged**